docs/docs/react-rehydration.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Understanding React Rehydration
+---
+
+One of the central ideas of Gatsby is that HTML content is statically generated--using React DOM server-side APIs. A follow up key feature is that this static HTML content can then be _enhanced_ with client-side JavaScript via React hydration which allows for [app-like features](/docs/adding-app-and-website-functionality/) in Gatsby sites.
+
+The steps taken to deliver a built site to the browser are discussed below:
+
+## Build and render static assets
+
+Running `gatsby build` starts up a Node.js server that processes your site: creating a GraphQL schema for your site, fetching data that your pages will pull in by [extracting queries](/docs/query-extraction/) from your code and [executing them](/docs/query-execution/), and [rendering each page's HTML](/docs/html-generation/).
+
+_The ["Overview of the Gatsby Build Process" conceptual guide](/docs/overview-of-the-gatsby-build-process/) helps explain what's happening at each step in the build process_
+
+All of this data is gathered during the build and written into the `/public` folder. You can [customize Gatsby's configurations](/docs/customization/) for `babel`, `webpack`, as well as the HTML generated by your build in order for finer grained tweaks in regards to how your site gets built.
+
+## Invoke the `ReactDOM.hydrate` method
```

Comment:
I miss here the info: is this done by Gatsby or should I need to place this command in my files